### PR TITLE
Acceptance test done and unit tests updated

### PR DESCRIPTION
**Pull request by JPeterson9**
Section acceptance test added and all unit tests can now be run as a directory. All unit tests now pass as well

### CODE REVIEW

**Review by turkogl3 — APPROVED** [review]

**Review by miladmasoodi — APPROVED** [review]
So I approved to update the names but the add_section acceptance tests and several other acceptance tests aren't testing anything other than the database and need to be changed

**Review by vitky25 — APPROVED** [review]

File: acceptance_tests/test_Add_Section.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        sections = SectionModel.objects.all()
+
+        self.assertEqual(len(sections), 2)
+
+    def test_course_invalid_add(self):
+        response = self.client.post('/', {'email': self.user.email, 'password': self.user.password})
+        self.assertEqual(response.url, '/home/')
+
+        resp = self.client.post(f'/addSection/', {'id': -3}, follow=True)
```
Comment (miladmasoodi) [inline acceptance_tests/test_Add_Section.py]: not a valid url needs id number and another slash and the actual inputs

File: acceptance_tests/test_Add_Section.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        self.section = SectionModel.objects.create(course_id=2, section_id=801)
+        self.user = MyUserModel.objects.create(user_id=1, email='Instructor@test.com', password='pass1234', name='Test',
+                                               address='USA', phone_number='123-456-7890', role=1)
+
+    def test_section_adding(self):
+        response = self.client.post('/', {'email': self.user.email, 'password': self.user.password})
+        self.assertEqual(response.url, '/home/')
+
+        self.course2 = CourseModel.objects.create(course_id=3, dept_code="COMP SCI", name=" Intro to AI")
+        self.section2 = SectionModel.objects.create(course_id=3, section_id=901)
```
Comment (miladmasoodi) [inline acceptance_tests/test_Add_Section.py]: shouldnt add with the database, it should update from the self.client.post(~~~~~)

File: acceptance_tests/test_Add_Section.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+        resp = self.client.post(f'/addSection/{self.section2.section_id}', {'id': self.section2.section_id})
```
Comment (miladmasoodi) [inline acceptance_tests/test_Add_Section.py]: the id in the url should be the course's id,

**Review by miladmasoodi — CHANGES_REQUESTED** [review]

**Pull request merged**